content/en/docs/plugin-guide/plugin-policy-engine.md

```diff
@@ -9,7 +9,9 @@ description: >
 
 ## Overview
 
-Armory's Policy Engine plugin is the next iteration of the Policy Engine feature that ships with our Armory distribution. Not only does it include support for existing features like applying policy to pipelines as they're being saved, it also introduces policy hooks into other services like Gate and Orca. 
+Armory's Policy Engine plugin is the next iteration of the Policy Engine feature that ships with our Armory distribution. Not only does it include support for existing features like applying policy to pipelines as they're being saved, it also introduces policy hooks into other services like Gate, the API, and Orca, the orchestrator. 
```

Comment:
When using commas to list I would suggest using parentheses to expand, to avoid awkward reading cadence.